demo/mobile/src/app/services/userService.js

```diff
@@ -19,29 +19,11 @@ UserService.prototype.readUser = function readUser(userId) {
   });
 };
 
-UserService.prototype.getProfile = function($http, $window) {
```

Comment:
Why we removed that method. We using this repository pattern and user related actions (listing and getting profile may be here. Obviously we should route that to active interface.